dev/io.openliberty.transport.http_fat/fat/src/io/openliberty/transport/http_fat/TcpOptionsDefaultTests.java

```diff
@@ -145,4 +145,15 @@ public void testHostNameIncludeList_default() throws Exception {
         // Validate that hostNameIncludeList default null.
         assertNotNull("The default value of hostNameIncludeList was not null!", server.waitForStringInTrace("hostNameIncludeList: null"));
     }
+
+    /**
+     * The test will check the default value of inactivityTimeout by searching the trace file.
+     *
+     * The default value is 60 seconds will be logged as 60000 miliseconds.
+     */
+    @Test
+    public void testInactivityTimeout_default() throws Exception {
+        // Validate that inActivityTimeout default is 60s.
+        assertNotNull("The default value of inactivityTimeout was not: 60000!", server.waitForStringInTraceUsingMark("inactivityTimeout: 60000"));
```

Comment:
Would it be worth printing out what the inactivityTImeout resolved to ? I know you are specifically grepping for 60000, so would need to adjust the string search, but wondering if there's value to log that. The server trace might show it regardless if trace is enabled.

Reply:
It is in the server trace which is being searched. I'd like to keep these tests simple I think the likely hood of us finding a failure here is very slim and mainly used to ensure behavior between netty and channel is the same.